### PR TITLE
Add documentation to cover Firefox CSP SVG sprite blocking issue

### PR DESCRIPTION
**Pull request by chrisdavidmills**
### Description

This PR adds documentation covering the Firefox CSP `default-src 'none'` SVG sprite blocking issue, documented by @pepelsbey at https://github.com/mdn/mdn-http-observatory/issues/6.

- I've added a bullet point to https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/CSP
- I've also added an example section to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src#examples so there is a direct section to link to.

### CODE REVIEW

**Review by pepelsbey — APPROVED** [review]
Looks good! I also double-checked how Firefox treats inlined sprites: it works! It might be an alternative solution if `default-src 'none'` is essential to keep. What do you think?

File: files/en-us/web/http/headers/content-security-policy/default-src/index.md [hunk 1]
```diff
@@ -79,6 +79,26 @@ Content-Security-Policy: connect-src 'self';
...
+
+However, there is a problem here. If you are embedding SVG sprites defined in external files via the [`<use>`](/en-US/docs/Web/SVG/Element/use) element, for example:
+
+```svg
+<svg>
+  <use href="/images/icons.svg#icon"/>
+</svg>
+```
+
+your SVG images will be blocked in Firefox if you have a `default-src 'none'` policy set. Firefox does not treat the SVG as an embedded image like other browsers do, therefore `img-src 'self'` will not allow them to be loaded. You need to use `default-src 'self'` if you want your external sprites to load in Firefox.
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/http/headers/content-security-policy/default-src/index.md]: Thanks @pepelsbey, this is great information. I have added it, but with slightly different wording/structure. My reasons:

1. If we refer to external sprites being inlined, then I reckon people will nitpick that they are no longer external.
2. I wanted to keep the bug/spec issue links attached to the bit about `default-src 'none'` not working with external SVG sprites specifically.

Let me know if you think my additions read ok.

**Review by pepelsbey — APPROVED** [review]
Perfect!

**Comment by chrisdavidmills** [comment]
Awesome, I've put it live, as I don't think there is any point waiting.

**Pull request merged**